src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplate.java

```diff
@@ -14,6 +14,7 @@
 import com.nirima.jenkins.plugins.docker.launcher.DockerComputerLauncher;
 import com.nirima.jenkins.plugins.docker.strategy.DockerOnceRetentionStrategy;
 import com.nirima.jenkins.plugins.docker.utils.UniqueIdGenerator;
+
```

Comment:
Our code changes in `fillContainerConfig` don't need this extra blank line - I suspect the IDE put it here - please remove it.

Reply:
fixed along with 3920fd0e8c43f4b4f4c19f96ac2c14a06a9a9af0